src/autocomplete.js

```diff
@@ -338,11 +360,10 @@ class Autocomplete extends Component {
   //\\\\\\\\\\\\\\\\\\\\\\\\\\\\\ ITEM
 
   reset = () => {
-    this.setState(({selectedItem}) => ({
+    this.setState({
       isOpen: false,
       highlightedIndex: null,
-      value: this.getValue(selectedItem),
```

Comment:
Ohh that wasn't a mistake. I handle that in `selectItem` now. I actually need to handle `multiple` and `inputValue`.